test/test_common/simulated_time_system.h

```diff
@@ -120,7 +124,8 @@ class SimulatedTimeSystemHelper : public TestTimeSystem {
   SystemTime system_time_ ABSL_GUARDED_BY(mutex_);
   TestRandomGenerator random_source_ ABSL_GUARDED_BY(mutex_);
   AlarmSet alarms_ ABSL_GUARDED_BY(mutex_);
-  std::map<Alarm*, AlarmSet::const_iterator> alarm_registrations_map_ ABSL_GUARDED_BY(mutex_);
+  std::unordered_map<Alarm*, AlarmSet::const_iterator>
```

Comment:
any reason not to use absl::flat_hash_map?

Reply:
Done.